rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -1970,9 +1970,11 @@ rmw_connextdds_message_info_from_dds(
   to->publication_sequence_number =
     static_cast<uint64_t>((from->publication_sequence_number).high) << 32 |
     static_cast<uint64_t>((from->publication_sequence_number).low);
+#if RMW_CONNEXT_DDS_API == RMW_CONNEXT_DDS_API_PRO
   to->reception_sequence_number =
     static_cast<uint64_t>((from->reception_sequence_number).high) << 32 |
     static_cast<uint64_t>((from->reception_sequence_number).low);
+#endif  // RMW_CONNEXT_DDS_API == RMW_CONNEXT_DDS_API_PRO
```

Comment:
In the `#else` case here, do we need to set `to->reception_sequence_number` to 0 explicitly?

Reply:
Good catch, it should be set to [RMW_MESSAGE_INFO_SEQUENCE_NUMBER_UNSUPPORTED](https://github.com/ros2/rmw/blob/01d0db60c5bac242fe6f8a6d49494512f1e20c61/rmw/include/rmw/types.h#L525).

Reply:
Fixed in 0c88722